### PR TITLE
Added kombu.transport.pyamqp import to pyinstaller

### LINKED ISSUE #119: Segmentation fault using the binary
Hello, I tried to execute the `celery-exporter` binary and I get a Segmentation fault ...

Perhaps `pip install` like is better than distributing a PyInstaller binaries ?

### PR DESCRIPTION
**Pull request by chripf**
This is required for work with rabbitmq.
Should fix #119 - regarding the last comments.

### CODE REVIEW

**Comment by CFarcy** [comment]
Any news on this PR being merged ?

**Comment by Legrems** [comment]
Dump? Still got the issue when using the executable alone :)

**Comment by danihodovic** [comment]
@CFarcy and @Legrems can you check out this branch and confirm that it works? Once you do I'll merge the PR.

Sorry about the delay, I was traveling through the jungle

File: Dockerfile.pyinstaller [hunk 1]
```diff
@@ -21,4 +21,5 @@ RUN eval "$(pyenv init -)" && pyinstaller cli.py -y --onefile --name celery-expo
         --hidden-import=celery.loaders.app \
         --hidden-import=celery.app.amqp \
         --hidden-import=celery.app.control \
-        --hidden-import=kombu.transport.redis
+        --hidden-import=kombu.transport.redis\
```
Comment (CFarcy) [inline Dockerfile.pyinstaller:24]:
```suggestion
        --hidden-import=kombu.transport.redis \
```

**Comment by CFarcy** [comment]
Thanks for your reply @danihodovic , I hope you had a good time in the jungle :)

There is a typo in the `Dockerfile.pyinstaller` (missing space before `\`). I propose a change for it.

Once the typo fix, it seems to works (at least it run with no error).
```sh
docker build . -t celery-exporter-builder -f Dockerfile.pyinstaller --build-arg PYTHON_VERSION=$(cat .python-version)
container=$(docker run --rm -d celery-exporter-builder sleep 5) && docker cp $container:/app/dist/celery-exporter ./celery-exporter
./celery-exporter --broker-url=amqp://localhost:5672
```

The metrics are available on localhost:9808
image

**Comment by danihodovic** [comment]
Released as v0.10.4 here https://github.com/danihodovic/celery-exporter/releases/tag/v0.10.4.

Thank you

**Pull request merged**